model.py

```diff
@@ -53,6 +53,9 @@ def __init__(self, json, username, base_url, abort_at):
         self.white_starts = self.initial_fen == "startpos" or self.initial_fen.split()[1] == "w"
         self.abort_at = abort_at
 
+    def is_abortable(self):
+        return len(self.state["moves"]) < 6
```

Comment:
Can we make "6" a config variable or define a global. No Magic numbers

Reply:
I don't see the harm of it. It's based on the UCI format, which doesn't allow moves longer than 5 chars.

If we don't want that, then we also don't want to split `moves` with a space char, because it's just a similar assumption. Are we also going to put the space in config or global?